conf/remote_url_conf.py

```diff
@@ -0,0 +1,4 @@
+#The URLs for connecting to BrowserStack and Sauce Labs.
+
+browserstack_url = "http://hub-cloud.browserstack.com/wd/hub"
+saucelabs_url = "https://ondemand.eu-central-1.saucelabs.com:443/wd/hub"
```

Comment:
Can you remove the port number from the url?
Using `HTTPS` will make it use `443`, we can skip adding the port number in the URL

Reply:
We could create this as new issue